scripts/shell_completions/bash/borg

```diff
@@ -71,40 +71,43 @@ _borg()
+                local opts="-n --dry-run -s --stats --list --filter --json --no-cache-sync --stdin-name --stdin-user --stdin-group --stdin-mode --content-from-command --paths-from-stdin --paths-from-command --paths-delimiter -e --exclude --exclude-from --pattern --patterns-from --exclude-caches --exclude-if-present --keep-exclude-tags --exclude-nodump -x --one-file-system --numeric-owner --noatime --noctime --nobirthtime --noflags --noacls --noxattrs --sparse --files-cache --read-special --comment --timestamp -c --checkpoint-interval --chunker-params -C --compression ${common_opts}"
                 ;;
             *' extract '*)
-                local opts="--list -n --dry-run --numeric-owner --nobsdflags --noacls --noxattrs --stdout --sparse -e --exclude --exclude-from --pattern --patterns-from --strip-components ${common_opts}"
+                local opts="--list -n --dry-run --numeric-ids --noflags --noacls --noxattrs --stdout --sparse -e --exclude --exclude-from --pattern --patterns-from --strip-components ${common_opts}"
                 ;;
             *' check '*)
-                local opts="--repository-only --archives-only --verify-data --repair --save-space --max-duration -P --prefix -a --glob-archives --sort-by --first --last ${common_opts}"
+                local opts="--repository-only --archives-only --verify-data --repair --save-space --max-duration -a --glob-archives --sort-by --first --last ${common_opts}"
                 ;;
             # rename
             #   no specific options
             *" list "*)
-                local opts="--short --format --json --json-lines -P --prefix -a --glob-archives --sort-by --first --last -e --exclude --exclude-from --pattern --patterns-from ${common_opts}"
+                local opts="--short --format --json-lines -a -e --exclude --exclude-from --pattern --patterns-from ${common_opts}"
                 ;;
             *' diff '*)
-                local opts="--numeric-owner --same-chunker-params --sort --json-lines -e --exclude --exclude-from --pattern --patterns-from ${common_opts}"
+                local opts="--numeric-ids --same-chunker-params --sort --json-lines -e --exclude --exclude-from --pattern --patterns-from ${common_opts}"
                 ;;
             *' delete '*)
-                local opts="-n --dry-run -s --stats --cache-only --force --save-space -P --prefix -a --glob-archives --sort-by --first --last ${common_opts}"
+                local opts="-n --dry-run --list --consider-checkpoints -s --stats --cache-only --force --keep-security-info --save-space -a --glob-archives --sort-by --first --last ${common_opts}"
```

Comment:
i recently removed `--save-space`, it didn't do anything since longer and was only kept for cmdline compat reasons.

needs fixing at other places also.

Reply:
Even the generated documentation still has `--save-space`. This needs more doing.

Reply:
Scratch that. I was looking at the 2.0.0b4 documentation.

Reply:
ok, i'll check it.